contracts/exchange/api/TransferContract.sol

```diff
@@ -219,6 +219,7 @@ abstract contract TransferContract is TradeContract {
 
     SubAccount storage toSubAcc = _requireSubAccount(toSubID);
     _requireSubAccountUnderAccount(toSubAcc, toAccID);
+    _fundAndSettle(toSubAcc);
```

Comment:
we missed a call to fundAndSettle here

Reply:
nit: might be clearer to do `_fundAndSettle` under `transfer`? 

State machine impl is:
```go
func (s *State) doTransfer(fromAccID common.Address, fromSubID uint64, toAccID common.Address, toSubID uint64, tokenCurrency types.Currency, numTokens uint64) error {
	if fromSubID != 0 {
		s.fundAndSettleSubAccountWithStateTime(s.SubAccounts[fromSubID])
	}
	if toSubID != 0 {
		s.fundAndSettleSubAccountWithStateTime(s.SubAccounts[toSubID])
	}
        // ...
	return nil
}
```